src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java

```diff
@@ -69,6 +77,22 @@ public class TestSetupHooks {
   public static String spannerTargetTable = StringUtils.EMPTY;
   public static boolean firstSpannerTestFlag = true;
   public static String datasetName = PluginPropertyUtils.pluginProp("dataset");
+  public static boolean firstBigTableTestFlag = true;
+  public static String bigtableInstance = StringUtils.EMPTY;
+  public static String bigtableCluster = StringUtils.EMPTY;
+
+  public static String bigtableSourceTable = StringUtils.EMPTY;
+  public static String bigtableTargetInstance = StringUtils.EMPTY;
+  public static String bigtableTargetCluster = StringUtils.EMPTY;
+
+  public static String bigtableTargetTable = StringUtils.EMPTY;
+  public static String bigtableExistingTargetTable = StringUtils.EMPTY;
+
+  public static BigtableInstanceAdminSettings instanceAdminSettings;
+  public static  BigtableInstanceAdminClient adminClient;
```

Comment:
nit: extra space

Reply:
Fixed,thanks.

Reply:
There is an extra space between `static` and `BigtableInstanceAdminClient`

Reply:
Fixed,thanks